layouts/dpg-standard/list.html

```diff
@@ -0,0 +1,6 @@
+{{ define "main" }}
+
+{{ partial "alert.html" . }}
+{{ partial "dpgindicators.html" . }}
```

Comment:
Ok👍

Reply:
I was wrong. This actually doesn't work. It seems a Hugo layout is only allowed to use one `{{ define "" }}` block while a partial can have many. We can regard this comment as work required for #121 and ignore the step for this PR. It will need to be addressed at the same time as #121.